kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -101,6 +102,8 @@
   public static final Long DEFAULT_WEBSOCKET_TIMEOUT = 5 * 1000L;
   public static final Long DEFAULT_WEBSOCKET_PING_INTERVAL = 1 * 1000L;
 
+  public static final Integer DEFAULT_MAX_CONCURRENT_REQUESTS_PER_HOST = 10;
```

Comment:
5 seems too low to me and it can cause hard to diagnose issues. So 10 seems to be a better default.